src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DetectPreviewFeatureTests.Interfaces.cs

```diff
@@ -37,12 +40,38 @@ public interface IProgram
                 void MarkedMethodInInterface();
             }
         }
-
             ";
 
             var test = TestCS(csInput);
             test.ExpectedDiagnostics.Add(VerifyCS.Diagnostic(DetectPreviewFeatureAnalyzer.ImplementsPreviewMethodRule).WithLocation(0).WithArguments("MarkedMethodInInterface", "IProgram.MarkedMethodInInterface", DetectPreviewFeatureAnalyzer.DefaultURL));
             await test.RunAsync();
+
+            var vbInput = @"
+        Imports System.Runtime.Versioning
+        Imports System
+
+        Namespace Preview_Feature_Scratch
+            Class Program
+                Implements IProgram
+
+                Private Shared Sub Main(ByVal args As String())
+                    Dim prog = New Program()
+                End Sub
+
+                Public Sub {|#0:MarkedMethodInInterface|}() Implements IProgram.MarkedMethodInInterface
```

Comment:
Could we apply the diagnostic like this instead?
```suggestion
                Public Sub MarkedMethodInInterface() Implements {|#0:Program.MarkedMethodInInterface|}
```

Or:
```suggestion
                Public Sub MarkedMethodInInterface() Implements IProgram.{|#0:MarkedMethodInInterface|}
```